users/details/script.js

```diff
@@ -58,6 +61,25 @@ async function getUserData() {
   }
 }
 
+function dataNotFound(message) {
```

Comment:
If this function, is reusable outside the scope of `user/details`, if yes then please move to utils and change the method to `getDataNotFound` and then add to the container.
else if not then no change required.

Reply:
I think @sahsisunny is only handling the Not Found for Users. Also a better name I could think of this function could be `generateNoUserFoundPage` or `generateNoDataFoundSection`

Reply:
change function name to `generateNoDataFoundSection`